src/commands/__tests__/login.test.ts

```diff
@@ -72,5 +81,14 @@ describe("login", () => {
       await login({ org: "test-org" });
       expect((signInWithCredential as jest.Mock).mock.calls).toMatchSnapshot();
     });
+    it("returns an error message if firebase cannot determine the user's email", async () => {
+      mockSignInWithCredential.mockResolvedValueOnce({
+        user: {},
+      });
+      await expect(login({ org: "test-org" })).rejects.toMatchInlineSnapshot(`
+"Can not sign in: this user has previously signed in with a different identity provider.
```

Comment:
This test is a little brittle, as any change to the message will cause the test to fail and require update. Is it not sufficient to verify that an exception is thrown?

Reply:
I'm trying to follow what we were already doing in this file.

```
await expect(login({ org: "test-org" })).rejects.toMatchInlineSnapshot(
    `"Could not find organization"`
);

await expect(login({ org: "test-org" })).rejects.toMatchInlineSnapshot(
    `"Unsupported login for your organization"`
    `"Unsupported login for your organization"`
);
```

Personally I like expecting on a particular message to ensure we are displaying the correct message to the user. Theoretically a logic change could expose an error with a different message.

Reply:
I think it's fine. It's only a `yarn jest -u` to update snapshots.